lib/krane.rb

```diff
@@ -1,5 +1,6 @@
 # frozen_string_literal: true
 
+require 'oj'
```

Comment:
Do we really need this? Do the tasks that are exported below really depend on `Oj` in any way?